tests/utils/test_ctftime.py

```diff
@@ -144,20 +141,14 @@ def test_ctf_ended():
     app = create_ctfd()
     with app.app_context():
         assert ctf_ended() is False
+        with CTFtime():
 
-        set_config(
-            "start", "1507089600"
-        )  # Wednesday, October 4, 2017 12:00:00 AM GMT-04:00 DST
-        set_config(
-            "end", "1507262400"
-        )  # Friday, October 6, 2017 12:00:00 AM GMT-04:00 DST
-
-        with freeze_time("2017-10-3"):
-            assert ctf_ended() is False
+            with freeze_time(FreezeTime.NOT_STARTED):
+                assert ctf_ended() is False
 
-        with freeze_time("2017-10-5"):
-            assert ctf_ended() is False
+            with freeze_time(FreezeTime.STARTED):
+                assert ctf_ended() is False
 
-        with freeze_time("2017-10-7"):
-            assert ctf_ended() is True
+            with freeze_time(FreezeTime.ENDED):
+                assert ctf_ended() is True
```

Comment:
I like the idea here but I think we can also keep going and create more context managers or functions so that we can setup more time context managers. 

What about something like:

```
with CTFTime.set():
    ...
    with CTFTime.started():
    ...
    with CTFtime.not_started():
    ...
    with CTFtime.ended():
```

Very rough code above, I didn't test it or think about it very much. Might work out better with a different structure.